package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "My-PhotoHub",
```

Comment:
We have 3 separate PRs that are all doing the same thing (creating `package.json`).  A better approach for doing this in the future would be to start with a very small PR that adds a `pacakge.json` and then build on that.  Open source should happen in small bite-sized chunks.

Reply:
So what should I do with the package.json?

Reply:
Moved the discussion to Slack